addons/radar/XEH_preInit.sqf

```diff
@@ -27,6 +27,7 @@ GVAR(oddDirectionCompasses) setVariable ["lib_ger_itemcompass", 6400];
 GVAR(maxDegrees) = 360;
 
 private _tfar = isClass (configFile >> "CfgPatches" >> "tfar_core");
+private _tfarOld = isClass (configFile >> "CfgPatches" >> "task_force_radio");
```

Comment:
Would say combine it with the other `_tfar` variable, no need to have two really if the events are the same

Suggestion:
```sqf
private _tfar = isClass (configFile >> "CfgPatches" >> "tfar_core") || {isClass (configFile >> "CfgPatches" >> "task_force_radio")};
```

Reply:
new TFAR also defines task_force_radio class, you can't use it to detect if you're running old version.
tfar_core missing means old

Reply:
@dedmen can I use ?
`getText (configfile >> "CfgPatches" >> "task_force_radio" >> "version") == "0.9.12"`

Reply:
This works:
```
private _tfar = isClass (configFile >> "CfgPatches" >> "tfar_core");
private _tfarOld = isClass (configFile >> "CfgPatches" >> "task_force_radio") && !_tfar;
```
Returns correct values depending on loaded version

Reply:
no you cannot check version